finance/pta/Portfile

```
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               perl5 1.0
+
+perl5.require_variant   yes
+
+name                    pta
+version                 0.1
+categories              finance
+platforms               {darwin any}
+supported_archs         noarch
+maintainers             @admacleod
```

A port should also be marked `openmaintainer` unless there is a reason why you and only you should be allowed to modify a port:

```suggestion
maintainers             {@admacleod} \
                        openmaintainer
```